fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1865,4 +1865,35 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         appContext.getPreferenceKey(R.string.pref_key_hidden_engines_restored),
         default = false,
     )
+
+    /**
+     * Indicates if the Firefox Suggest feature is enabled.
+     */
+    var enableFxSuggest by lazyFeatureFlagPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_enable_fxsuggest),
+        default = { FxNimbus.features.fxSuggest.value().enabled },
+        featureFlag = FeatureFlags.fxSuggest,
+    )
+
+    /**
+     * Indicates if the user has chosen to show sponsored search suggestions in the awesomebar.
+     * The default value is computed lazily, and based on whether the Firefox Suggest feature is
+     * enabled.
+     */
+    var showSponsoredSuggestions by lazyFeatureFlagPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_show_sponsored_suggestions),
+        default = { enableFxSuggest },
+        featureFlag = true,
```

Comment:
It shouldn't really matter since the settings for these are hidden by the featureFlag, but using the `FeatureFlags.fxSuggest` consistently for these might reduce future confusion. It could even potentially prevent errors if someone adds another way to update these settings.